0.data_analysis/plate_5_characteristics/nbconverted/plate_5_sc_feature_selected_camerons_agg_well_corr.py

```diff
@@ -41,11 +43,11 @@
 
 # Path to correlation class
 sys.path.append(
-    f"{root_dir}/0.data_analysis/plate_5_characteristics/utils"
+    f"{root_dir}/0.data_analysis/utils"
 )
 
 # Class for calculating correlations
-from CorrelatePlate import CorrelatePlate
+from CorrelateData import CorrelateData
 
 platedf_path = pathlib.Path(root_dir / "nf1_painting_repo/3.processing_features/data/single_cell_profiles/Plate_5_sc_feature_selected.parquet").resolve(strict=True)
```

Comment:
I see that below you perform "Cameron's method," but we already have data where this method was performed during pycytominer processing: https://github.com/WayScience/nf1_cellpainting_data/blob/main/3.processing_features/data/single_cell_profiles/Plate_5_bulk_camerons_method.parquet

Is there a reason you decide not to use that data?

Reply:
Force of habbit, it might be better to avoid the extra computation and use this data instead. I'll probably change this

Reply:
Changed